test/mlj_flux_interfacing.jl

```diff
@@ -0,0 +1,112 @@
+using Random: Random
+import Random.seed!
+using MLJ
+using MLJBase
+using MLJFlux
+using Flux
+using StableRNGs
+
+function basictest(ModelType, X, y, builder, optimiser, threshold, accel)
+    ModelType_str = string(ModelType)
+    ModelType_ex = Meta.parse(ModelType_str)
+    accel_ex = Meta.parse(string(accel))
+    optimiser = deepcopy(optimiser)
+
+    eval(
```

Comment:
why the `eval`?

Reply:
This is a bit of a remnant of me thinking that will have to make two models, using some meta programming and eval, we can convert the passed arguments into a julia expression and evaluate it. And thus the model can be created inside the function based on the name passed without conditionals or anything like that. This can certainly be removed since there is only one model now

Reply:
Yes, I believe this is better removed, since wouldn't `eval` cause compilation at run-time? I am not sure the compiler can optimize and extrapolate the values into the evaluated expression at compile-time.